source/playbooks/work-with-tasks.rst

```diff
@@ -0,0 +1,20 @@
+Work with tasks
+===============
+
+Tasks and due dates
+-------------------
+
+In some workflows, there are time constraints on tasks and others may have more flexible timeframes. Associating tasks with deliverable dates provides visibility into workloads and helps everyone stay accountable during the run.
+
+To assign a due date to a task, select the **Toggle Run Details** icon to open the **Run Details** screen. Hover over the task you’d like to edit and select the calendar icon to assign a due date. Due dates can be used to sort tasks in the run overview.
+
+When a due date is assigned to a task, and the task is overdue or due today, a reminder is added to the Playbooks daily digest along with tasks that don't have an assigned due date. As tasks are completed, they're removed from the daily digest reminders. You can refresh the list of assigned tasks at any time using the ``/playbooks to-do`` slash command.
+
+Due dates can be entered in text (e.g., "two minutes ago") or numerically (e.g., "15 March 2023").
+
+Task inbox
+----------
+
+In addition to the daily digest, you also have access to a task inbox. To view it, open the playbooks tab. 
+
+In the header, next to your profile photo, select the message icon. A list of every task assigned to you from every run that's in progress is displayed. You can check off, change due dates, and reassign tasks in this view.
```

Comment:
The tasks that are shown in task inbox are:
- pending & assigned to me tasks
- pending & unassigned tasks from runs where I'm the owner

Once it's shown, the user can change the filtering to:
- show only tasks that are explicitly assigned to him
- show also checked tasks

The sorting order will be based on the due date if exists